settings.js

```diff
@@ -13,4 +13,10 @@ const spotlightEmoji = '⭐';
 // Allow twitch emotes, can be true or false.
 const twitchEmotes = true;
 
+// Play a sound when the "!alert" command is used
+const isAlertSound = false;
```

Comment:
Just so variable make sense could this be 'playAlertSound?'

Reply:
Done!